packages/beacon-node/src/node/nodejs.ts

```diff
@@ -161,7 +161,7 @@ export class BeaconNode {
     metricsRegistries = [],
   }: BeaconNodeInitModules): Promise<T> {
     if (hasher.name !== "hashtree") {
-      throw Error(`Loaded incorrect hasher ${hasher.name}, expected hashtree`);
+      logger.warn("hashtree is not supported, using hasher", {hasher: hasher.name});
```

Comment:
this would print hasher twice, might wanna inline variable in that case

```suggestion
      logger.warn(`hashtree is not supported, using hasher ${hasher.name}`);
```